pipenv/cli/options.py

```diff
@@ -126,7 +126,7 @@ def callback(ctx, param, value):
         state.installstate.skip_lock = value
         return value
     return option("--skip-lock", is_flag=True, default=False, expose_value=False,
-                    help=u"Ignore locking mechanisms when installing—use the Pipfile, instead.",
+                    help=u"Ignore locking mechanisms during operation—use the Pipfile, instead.",
```

Comment:
looking at this now it seems like "ignoring" might be the wrong way to word this, since we aren't exactly ignoring it at all.  It's not like we are pretending the operation occurred, but in reality it didn't occur -- we are intentionally skipping it. 

Maybe we can say something like "Skip dependency resolution and locking and rely on Pipfile and package installation order instead." Is that too detailed? Idk, maybe this is nitpicky

Reply:
Exactly. I've updated it by emphasizing skip instead of ignore and also keep the info text short. What do you think about it now?